cmd/logger/logger.go

```diff
@@ -53,61 +62,73 @@ func (level Level) String() string {
 	return lvlStr
 }
 
-type logEntry struct {
-	Level   string   `json:"level"`
-	Message string   `json:"message"`
-	Time    string   `json:"time"`
-	Cause   string   `json:"cause"`
-	Trace   []string `json:"trace"`
+type traceEntry struct {
+	Message   string            `json:"message"`
+	Source    []string          `json:"source"`
+	Variables map[string]string `json:"variables,omitempty"`
+}
+type args struct {
+	Bucket string `json:"bucket,omitempty"`
+	Object string `json:"object,omitempty"`
 }
 
-// Logger - for console messages
-type Logger struct {
-	quiet bool
-	json  bool
+type api struct {
+	Name string `json:"name,omitempty"`
+	Args args   `json:"args,omitempty"`
 }
 
-// NewLogger - to create a new Logger object
-func NewLogger() *Logger {
-	return &Logger{}
+type logEntry struct {
+	Level      string     `json:"level"`
+	Time       string     `json:"time"`
+	API        api        `json:"api,omitempty"`
+	RemoteHost string     `json:"remotehost,omitempty"`
+	RequestID  string     `json:"requestID,omitempty"`
+	UserAgent  string     `json:"userAgent,omitempty"`
+	Cause      string     `json:"cause,omitempty"`
+	Trace      traceEntry `json:"error"`
 }
 
+var (
+	quiet, jsonFlag bool
+)
+
 // EnableQuiet - turns quiet option on.
-func (log *Logger) EnableQuiet() {
-	log.quiet = true
+func EnableQuiet() {
+	quiet = true
 }
 
 // EnableJSON - outputs logs in json format.
-func (log *Logger) EnableJSON() {
-	log.json = true
-	log.quiet = true
+func EnableJSON() {
+	jsonFlag = true
+	quiet = true
 }
 
 // Println - wrapper to console.Println() with quiet flag.
-func (log *Logger) Println(args ...interface{}) {
-	if !log.quiet {
+func Println(args ...interface{}) {
+	if !quiet {
 		console.Println(args...)
 	}
 }
 
 // Printf - wrapper to console.Printf() with quiet flag.
-func (log *Logger) Printf(format string, args ...interface{}) {
-	if !log.quiet {
+func Printf(format string, args ...interface{}) {
+	if !quiet {
 		console.Printf(format, args...)
 	}
 }
 
-func init() {
+// Init :
```

Comment:
we should write a proper comment here... we don't need to ignore

Reply:
done